scheduler/scheduler-client/src/main/java/org/ow2/proactive/scheduler/common/util/SchedulerProxyUserInterface.java

```diff
@@ -422,7 +422,12 @@ public boolean pauseJob(JobId jobId) throws NotConnectedException, UnknownJobExc
     public boolean removeJob(JobId jobId) throws NotConnectedException, UnknownJobException, PermissionException {
         checkSchedulerConnection();
         return uischeduler.removeJob(jobId);
+    }
 
+    @Override
+    public boolean removeJobs(List<JobId> jobIds) throws NotConnectedException, PermissionException {
```

Comment:
No more  UnknownJobException exception management ?

Reply:
Exactly. It was by design. Fabien asked for it.

Reply:
Additionnally, regarding PermissionException. What if one job cannot be removed due to permission? Is the request still applied to other jobs (notably jobids which are after the offending id)?

Reply:
Currently, one permission exception would abort whole operation